Lombiq.NodeJs.Extensions/pnpm-lock.yaml

```diff
@@ -1,5 +1,6639 @@
-lockfileVersion: '6.0'
+lockfileVersion: 5.4
```

Comment:
Why the downgrade form 6.0? The root one in OSOCE is 6.0.

Reply:
I think it must've been a merge mistake. I've regenerated it.